test/plugins/fingerprint.pulgin.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { VerifyFiltersDto } from '../../src/plugins/dto/verify.filters.dto';
+import { FingerprintPlugin } from '../../src/plugins/impl/fingerprint.plugin';
+import { MockBioAuthService } from '../mock/mock.bio.auth.service';
+import { ExternalId } from '../../src/db/entity/external.id';
+import { MockRepository } from '../mock/mock.repository';
+import { VerifyFingerprintImageDto } from '../../src/plugins/dto/verify.fingerprint.image.dto';
+import { ProtocolException } from 'protocol-common/protocol.exception';
+import { ProtocolErrorCode } from 'protocol-common/protocol.errorcode';
+
+const params = {
+    position: 1,
+    image: 'some image'
+} as VerifyFingerprintImageDto;
+const filters = {
+    externalIds: {
+        key: 'value'
+    }
+} as VerifyFiltersDto;
+const agentId = 'agentId123';
+
+describe('Test to see if test status is other than "matched" will cause ProtocolException to throw', () => {
+    //const fingerprintVerifySpec = new VerifyFiltersDto ;
+    it('Plugin is not empty', async () => {
+        const bioAuthService = new MockBioAuthService('not_matched', agentId);
+        const mockExternalId = new ExternalId();
+        mockExternalId.agent_id = agentId;
+        mockExternalId.external_id = "foobar";
```

Comment:
Please use 'single-quotes' instead of "double-quotes". I thought the linter would catch this... I suspect it's not working on test code for some reason...